zipline/utils/calendars/trading_calendar.py

```diff
@@ -684,17 +685,22 @@ def minute_to_session_label(self, dt, direction="next"):
             The label of the containing session.
         """
 
-        idx = searchsorted(self.market_closes_nanos, dt.value)
+        if type(dt) is pd.Timestamp:
```

Comment:
Unless subclasses aren't allowed, what do you think of `isinstance` instead?

Could be a good case for something like `@preprocess(dt=coerce(pd.Timestamp, attrgetter('value')))`.